orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/SomaticFindingsChapter.java

```diff
@@ -395,4 +415,26 @@ private static <T> List<T> max10(@NotNull List<T> elements)
     {
         return elements.subList(0, Math.min(10, elements.size()));
     }
+
+    private static boolean hasHighDriverEvents(@NotNull List<com.hartwig.hmftools.datamodel.linx.LinxFusion> somaticFusions,
```

Comment:
I don't think you need the full package here for LinxFusion?

Reply:
Yes you're right, happened because I moved this function (first it used the other `LinxFusion`)